app/scripts/directives/navBarMenuCloseable.dir.js

```diff
@@ -0,0 +1,22 @@
+ 'use strict';
+
+  angular.module('yourStlCourts').directive('navBarMenuCloseable', function($document){
+    return{
+      restrict: 'C',
```

Comment:
There is really no point in having a class directive. I think an attribute directive is more common and fits well in this particular scenario.